### ISSUE TITLE
Could not access source file: working on internal cache only

### ISSUE THREAD

**Issue by Larch35** [PhilippC/keepass2android#1216]
Hello. I'm using the latest version of keepass2android on Android v9 / EMUI 9.1.0, and trying to open my database which is stored on Google Drive.
When I try to open the file, an box pops up in the centre of the screen with the above error (and some additional information, but the message disappears and I can't screenshot it to attach here).
As a workaround, I have to save (download) the file manually to my device each time before opening in keepass2android, which seems to defeat the purpose of the 'online' version of the app (i.e. as opposed to keepass2android offline).
Any ideas, please? Let me know if you require any further info.
Thanks.

**Comment by PhilippC** [comment]
sounds like you have ticked the box to use the internal cache only?

**Comment by Larch35** [comment]
Uh, don't I feel like quite the idiot. :-/ I hadn't noticed that option, as it's obscured by the "Change Database / Unlock" toolbar, which floats up the screen when the keyboard is active (and the keyboard is active as soon as I launch the app). What is odd is that I only recently installed the app and don't recall selecting that option, but I'm inferring from your question that it is not enabled by default. However, I did have the offline app installed before that, so perhaps the setting was inherited from that? In any case, problem solved - sorry for the silly question. 😊 
Incidentally (and I hope this isn't another silly question), why _are_ there two versions of the app, if the standard app can effectively be used like the offline one anyway?
Thanks
